### DISCUSSION TITLE
Livewire V3 / Wire:navigate on links.

### DISCUSSION THREAD

**Discussion by ethanpatchell** [rappasoft/laravel-livewire-tables#1319] [Q&A]
When adding wire:navigate into the published view assets, it still reloads the page to send me to the route.

I've cleared the cache, view, routes and config but still not working.

Any ideas?

`<a href="{{ $path }}" wire:navigate {!! count($attributes) ? $column->arrayToAttributes($attributes) : '' !!}>{{ $title }}</a>`

`LinkColumn::make('Action')
                ->title(fn($row) => 'Edit')
                ->location(fn($row) => route('admin.pet-codes.manage', ['code'=>$row->code]))
                ->attributes(fn($row) => [
                    'class' => 'rounded-full',
                    'alt' => $row->name . ' Avatar',
                ]),`

**Comment by lrljoe** [comment]
There was a bug with Livewire 3 core around the time you posted this, but there are a number of tweaks and fixes in the pending PRs

By Wednesday these will all be in v3-master, but feel free to use v3-develop to get the latest fixes

**Comment by lrljoe** [comment]
This should be fixed in v3-develop.

Anyone other than @ethanpatchell  experiencing this issue at a later date, please open a new discussion rather than replying to this one.
@ethanpatchell  - feel free to reply if you still have the issue.,
